### PR TITLE
Fix PEP-8 issues in contributing.rst

### PR DESCRIPTION
**Pull request by FHaase**

### CODE REVIEW

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -963,7 +970,7 @@ a single test.
 
 .. code-block:: python
 
-   with warch.catch_warnings():
```
Comment (FHaase) [inline doc/source/contributing.rst]: I assume `warnings` was meant.
If someone knows better I can change it.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @FHaase 

I also think `warnings` was intended

**Comment by datapythonista** [comment]
@FHaase, I think you can merge master and leave the CI in green now.

Also, if you can remove the file from `setup.cfg`

**Comment by jreback** [comment]
thanks @FHaase

**Pull request merged**